mockfeaturesource.go

```diff
@@ -2,6 +2,7 @@ package main
 
 import "github.com/stretchr/testify/mock"
 
+// MockFeatureSource for testing support
```

Comment:
This file is auto-generated. Making manual changes to auto-generated files is not recommended. Please revert this change.